lms/templates/courseware/progress_graph.js

```diff
@@ -46,40 +57,47 @@ $(function () {
                                               'color' : colors[colorIndex]}
       
       categoryData = categories[ section['category'] ]
-    
+
+      ## Because this is Python (Mako) embedded in JavaScript, our safe linting script is
+      ## thoroughly confused. We should rewrite this file to remove Python/Mako.
+      ## safe-lint: disable=javascript-jquery-append
       categoryData['data'].append( [tickIndex, section['percent']] )
+      ## safe-lint: disable=javascript-jquery-append
       ticks.append( [tickIndex, section['label'] ] )
     
       if section['category'] in detail_tooltips:
+          ## safe-lint: disable=javascript-jquery-append
           detail_tooltips[ section['category'] ].append( section['detail'] )
       else:
           detail_tooltips[ section['category'] ] = [ section['detail'], ]
           
       if 'mark' in section:
+          ## safe-lint: disable=javascript-jquery-append
           droppedScores.append( [tickIndex, 0.05] )
+          ## safe-lint: disable=javascript-jquery-append
           dropped_score_tooltips.append( section['mark']['detail'] )
         
       tickIndex += 1
     
       if section.get('prominent', False):
           tickIndex += sectionSpacer
           
-  ## ----------------------------- Grade overviewew bar ------------------------- ##
+  ## ----------------------------- Grade overview bar ------------------------- ##
   tickIndex += sectionSpacer
   
   series = categories.values()
   overviewBarX = tickIndex
   extraColorIndex = len(categories) #Keeping track of the next color to use for categories not in categories[]
   
-  if show_grade_breakdown:    
+  if show_grade_breakdown:
     for section in grade_summary['grade_breakdown']:
         if section['percent'] > 0:
             if section['category'] in categories:
                 color = categories[ section['category'] ]['color']
             else:
                 color = colors[ extraColorIndex % len(colors) ]
                 extraColorIndex += 1
-        
+            ## safe-lint: disable=javascript-jquery-append
```

Comment:
Rather than `$.append` you can use [`HtmlUtils.append`](http://ui-toolkit.edx.org/utilities/html-utils/#htmlutilsappendelement-html)